README.md

```diff
@@ -8,12 +8,41 @@ The TACC CORE-CMS can be run using Docker and Docker Compose, both locally and i
 
 ### Required
 
-1. Create a `.env` at the root of the project, with the content `CUSTOM_ASSET_DIR=example-cms`.
-1. Initialize submodules and retireve the latest relevant code.
-    1. `git submodule init` (only necessary once)
-        - Add `cms-site-resources` repo as `taccsite_custom/`.
-    2. `git submodule update`
-        - Get latest desired submodule commit (this adds directory content).
+1. Create a `.env` with this content\* at the root of the project:
+
+    ```bash
+    CUSTOM_ASSET_DIR=example-cms
+    ```
+
+    \* Where `example-cms` is the project to be run. _See [(Optional) Custom Resources per CMS Project](#optional-custom-resources-per-cms-project)._
+1. Initialize / Update submodules.
+    1. `git submodule init`\
+        _(Adds `cms-site-resources` repo as submodule at `taccsite_custom/`. Only necessary once per `CORE-cms` repo clone.)_
+    2. `git submodule update`\
+        _(Downloads code from pinned commit of `cms-site-resources` repo to `taccsite_custom/`.)_
+    3. (as necessary) Change the pinned commit of submodule.\
+        _(Downloads code from different commit of `cms-site-resources` repo to `taccsite_custom/`.)_
+
+        ```bash
+        # Navigate into submodule (from root of this repo)
+        cd taccsite_custom
+        # Checkout different commit of submodule repo
+        git checkout master_or_other_branch_or_commit
+        # Update commit of submodule repo (as necessary)
+        git pull # (or git fetch […], etc.)
+        ```
+    4. (as necessary) Save/Commit the pinned commit of submodule.
+        _(Updates the pinned commit of `cms-site-resources` repo to `taccsite_custom/`.)_
+
+        ```bash
+        # Navigate back to this repo (from root of submodule repo)
+        cd ../
+        # Commit this repo's pointer to a different commit of submodule repo
+        git add taccsite_custom
+        git commit -m "[…] Update submodule to get […]"
+        ```
+
+<!-- IDEA: For brevity, consider moving steps 3 and 4 to a different section or document -->
```

Comment:
_Lot's of text…_

## Idea(s)

Move this to the "(Optional) Custom Resources per CMS Project" section, to allow it to be more easily skipped. but, should it be more easily skipped? Or, should all submodule custom project setup be in its own doc?

Reply:
Excellent! For now I think this works as it captures the whole process clearly. 
As we iron out the deployment details with Camino these details may eventually change some, ut works for the time being.